javascript/validation.js

```diff
@@ -0,0 +1,73 @@
+// input값 가져오기
+const ID_INPUT = document.getElementById("username");
+const PW_INPUT = document.getElementById("password");
+const LOGIN_BUTTON = document.getElementById("login_btn");
+
+// 에러
+function idcheck(ID_INPUT){ // id 에러 상황
+  if(ID_INPUT.value === ''){
+    showidErrorMessage("이메일을 입력해 주세요.");
+  }
+  else if(!isValidEmail(ID_INPUT.value)){
+    showidErrorMessage("올바른 이메일 주소가 아닙니다.");
+  }
+} 
+
+function pwcheck(PW_INPUT){ // pw 에러 상황
+  if(PW_INPUT.value === ''){
+    showpwErrorMessage("비밀번호를 입력해 주세요.");
+  }
+}
+
+// 이메일유효성 검사
+function isValidEmail(email) {
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+  return emailRegex.test(email);
+}
+
+
+// 페이지이동(fqa)
+function login(ID_INPUT, PW_INPUT){ //아이디:codeit@codeit.com, 비밀번호:codeit101 일때
+  if(ID_INPUT.value === "codeit@codeit.com" && PW_INPUT.value === "codeit101"){
```

Comment:
보통 이런 경우 test 계정을 상수로 선언해두고 여기서는 가져다 쓰면 실수를 미연에 방지할 수 있습니다.
const TEST_EMAIL = 'test@codeit.com'
const TEST_PW = 'codeit101'

```suggestion
  if(ID_INPUT.value === TEST_EMAIL && PW_INPUT.value === TEST_PW) {
```